stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>Super minor but I find the <code>list(set().union(...))</code> line a bit weird. I tried playing with it and ended up with this, but I'm not sure it's any clearer tbh </p><pre class="ql-syntax" spellcheck="false">team_winners = set(owl21_reduced["map_winner"])
team_losers = set(owl21_reduced["map_loser"])
team_names = list(team_winners | team_losers)
team_names.remove("draw")
team_names
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Since they're not used elsewhere, I guess at least the <code>unique()</code> operators on <code>team_winners</code> and <code>team_losers</code> aren't really needed.</p><p> </p>

Reply:
<p>Yeah, that's much nicer, thanks.</p>